indico/modules/rb_new/client/js/common/timeline/DailyTimelineContent.jsx

```diff
@@ -290,10 +292,20 @@ export function TimelineRowLabel({label, availability, longLabel, onClickLabel})
             color = 'green';
             tooltip = Translate.string('Room available');
     }
+
+    const labelContent = verboseName ? (
+        <span styleName="split-label">
+            <div>{name}</div>
+            <div styleName="sub-label">
+                <small>{verboseName}</small>
```

Comment:
Maybe it's worth trying [`<TooltipIfTruncated />`](https://github.com/indico/indico/blob/master/indico/web/client/js/react/components/TooltipIfTruncated.jsx) here in case the `verboseName` is too long that it gets _ellipsis'ed_.

Reply:
Does the `<TooltipIfTruncated>` that wraps around the label <div> also apply here?

Reply:
Ah, I didn't notice it. But it seems that the `<TooltipIfTruncated />` is only working for the `name` and not for the `verboseName`.

I also noticed this.
![image](https://user-images.githubusercontent.com/8863238/53750976-800a5000-3eab-11e9-8312-3ac5f189dcdb.png)
The `name` is not being _ellipsized_ as it used to be that might be ugly if there are too long room names (I'm not sure if there are)

Reply:
Good observation but I think the name is always going to be in the format of: Building/Floor-RoomNo right?

Reply:
I think so, so maybe it's worth only applying the tooltip for the `verboseName`. Let's ping the others to see what they think. @pferreir @ThiefMaster @mic4ael @meluru

Reply:
I think it's a good idea.